### PR TITLE
Improve recognition of files to be processed and archived

### PR DESCRIPTION
**Pull request by kjsanger**
Add support for TSV files.

Add support for compressing CSV files.

Use path.UncompressedFilename() more liberally to streamline adding
compression to other file types.

Add new all ONT-specific paths to the exclusion list.

Add the /data/laboratory WSI-specific path to the exclusion list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,18 +26,20 @@ these types with a simple gzip.
...
 #### Archiving files
 
 - Files patterns supported
 
-  - *.csv$
-  - *.fast5$
-  - *.gz$
-  - *.md$
-  - *.pdf$
+  - `*.csv`
```
Comment (mksanger) [inline README.md:37]: Should this be removed, since `*.csv` is only copied if compressed (valet/predicates.go#L69)?

Comment (kjsanger) [reply README.md:37]: I think you're right that the lists aren't consistent - I intended the two lists to be "files that will be compressed" and "files that are supported", so really I need to add `fastq` and `txt` to the second list.

File: README.md [hunk 2]
```diff
@@ -21,23 +21,27 @@ disks filling.
 #### Compressing files
 
 Some versions of MinKNOW to do not compress fastq files and no version
```
Comment (dkj) [inline README.md:23]:
```suggestion
Some versions of MinKNOW do not compress fastq files and no version
```

**Review by dkj — APPROVED** [review]
Curious that we compress csv but not tsv - presume it's the size of such files seen so far

**Comment by kjsanger** [comment]
TSV not compressed because so far they have been tiny.

**Pull request merged**